projects/plugins/jetpack/_inc/client/my-plan/my-plan-body.jsx

```diff
@@ -296,6 +327,7 @@ class MyPlanBody extends React.Component {
 						{ 'is-premium-plan' === planClass && getRewindVaultPressCard() }
 						{ 'is-business-plan' === planClass && getRewindVaultPressCard() }
 						{ this.props.hasInstantSearch && getSearchCard() }
+						{ 'is-jetpack-social-plan' === planClass && getSocialCard() }
```

Comment:
I'm not sure this is ever the case, as it's not actually a plan at this point. I think we'd need to take the same approach as search and check for the feature.

There are other blocks where this `hasInstantSearch` check is made too, so we might have to add it in there as well